chain/client/src/client_actor.rs

```diff
@@ -62,6 +63,9 @@ const BLOCK_HORIZON: u64 = 500;
 const HEAD_STALL_MULTIPLIER: u32 = 4;
 
 pub struct ClientActor {
+    /// Synchronization measure to allow graceful shutdown.
+    /// Informs the system when a ClientActor gets dropped.
+    _sender: Option<oneshot::Sender<()>>,
```

Comment:
Let's put this field last: rust drops struct fields in order, and we want this field to be dropped last to make sure that everything is fully drop by the time we get the signal.